docs/guides/software-lifecycle/security/security-scanning/README.md

```diff
@@ -0,0 +1,208 @@
+# Security Scanning
+
+<pre align="center">Guide to code scanning for security issues.</pre>
+
+## Introduction
+
+**Background**: Software security is critical in modern systems with application code at its root. Identifying and addressing vulnerabilities rapidly mitigates risk and limits the potential surface area of attacks. We recommend [NASA's SCRUB platform](https://github.com/nasa/scrub) to manage code scanning by identifying, orchestrating and aggregating security information. SCRUB's GitHub implementation wraps [CodeQL](https://codeql.github.com/) results into compact, curated reports that highlight security assessments and are suitable for ingestion by automated reporting tools. A small configuration is appended to an existing CodeQL configuration (`codeql-config.yml` file) that specifies security analyses and reporting properties.
```

Comment:
I prefer the Oxford or serial comma—but that's a personal taste issue: `…identifying, orchestrating, and aggregating…`

Reply:
Thanks. I'm living in an old AP Style Guide world from a previous career, or is it Strunk and White? I'm fine either way!